IDE/VS-AZURE-SPHERE/server/server.c

```diff
@@ -0,0 +1,194 @@
+﻿/* server.c
+ *
+ * Copyright (C) 2006-2019 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL. (formerly known as CyaSSL)
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+#include "server.h"
+/* the usual suspects */
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+/* socket includes */
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <unistd.h>
+
+/* wolfSSL */
+#include <wolfssl/ssl.h>
+#include <wolfssl/certs_test.h>
+
+/* Azure Sphere */
+#include <applibs/log.h>
+#include <applibs/networking.h>
+
+#define DEFAULT_PORT    11111
+#define CERT_BUF        server_cert_der_2048
+#define SIZEOF_CERT_BUF sizeof_server_cert_der_2048
+#define KEY_BUF         server_key_der_2048
+#define SIZEOF_KEY_BUF  sizeof_server_key_der_2048
+
+int main(void)
+{
+    bool isNetworkingReady = false;
+    int                sockfd;
+    int                connd;
+    struct sockaddr_in servAddr;
+    struct sockaddr_in clientAddr;
+    socklen_t          size = sizeof(clientAddr);
+    char               buff[256];
+    size_t             len;
+    int                shutdown = 0;
+    int                ret;
+    const char*        reply = "I hear ya fa shizzle!\n";
+
+    /* declare wolfSSL objects */
+    WOLFSSL_CTX* ctx;
+    WOLFSSL*     ssl;
+
+    /* Check if the Azure Sphere Dev Board has network connectivity. */
+    if ((Networking_IsNetworkingReady(&isNetworkingReady) < 0) || !isNetworkingReady) {
```

Comment:
Can we print the server's IP address to simply testing with a client?

Reply:
Ya, I'll work on that.  Looks like it's possible to get all the interfaces and their IP address using getifaddrs.  There are some other alternatives to.